_includes/js/sorttable.js

```diff
@@ -0,0 +1,149 @@
+class SortableTable {
```

Comment:
OBO Foundry has struggled with code maintainability for a long time. It's imperative that this code is well-documented and that a linter is applied (I'm not super familiar with JS linters but I think [prettier](https://prettier.io/) is a potential option). Could you please document all classes and functions using JavaScript standard style?

Reply:
Kenyi, can you include a script comment that references where this code came from? It would help if the code is documented in its original (github?) location.

Reply:
Will make the necessary changes and update the pull request.
#1972 and #1971 are already included, will make the change for #1970
as well